Error: 
  â”Œ tests/js_error/source_locations/expression/assign/set_private_property.js:3:11
  |
3 |     throw new Error()
  |           ^
Stack trace:
  at set #foo (tests/js_error/source_locations/expression/assign/set_private_property.js:3:11)
  at test (tests/js_error/source_locations/expression/assign/set_private_property.js:7:9)
  at <global> (tests/js_error/source_locations/expression/assign/set_private_property.js:12:1)